src/oscar/apps/checkout/mixins.py

```diff
@@ -268,44 +263,15 @@ def send_signal(self, request, response, order):
     def get_success_url(self):
         return reverse('checkout:thank-you')
 
-    def send_confirmation_message(self, order, code, **kwargs):
```

Comment:
Think we need to document that this method has been removed and replaced with `send_order_placed_email` - could you please add this to the release notes?

Reply:
Added to `docs/source/releases/v2.1.rst`